var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -126,6 +127,7 @@ class Trilinos(CMakePackage, CudaPackage, ROCmPackage):
     variant('tempus',       default=False, description='Compile with Tempus')
     variant('tpetra',       default=True, description='Compile with Tpetra')
     variant('trilinoscouplings', default=False, description='Compile with TrilinosCouplings')
+    variant('triutils',     default=False, description='Compile with Triutils')
```

Comment:
As my original review requested, these new variants need `conflicts` statements to attempt to mirror the TriBITS package logic in Spack (since all valid combinations of variants should produce a working binary, so that different libraries and apps can guarantee that they'll work together). Look at `amesos2` as an example.

Reply:
I still don't understand what the extra `triutils` variant adds to the package: does your code need it downstream? If so let's just turn it on by default if it doesn't require any upstream dependencies.